frontend/components/disclaimer.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 
+const flagUrl = require('uswds/dist/img/us_flag_small.png');
+
 const Disclaimer = () =>
   <div className="usa-disclaimer">
     <div className="usa-grid">
       <span className="usa-disclaimer-official">
         <img
           className="usa-flag_icon"
           alt="US flag signifying that this is a United States Federal Government website"
-          src="/images/uswds/us_flag_small.png"
+          src={flagUrl}
```

Comment:
Is this related to the travis-ci fail? I was expecting it to be from the change of base branch, but then I saw this:

```
/home/travis/build/18F/federalist/node_modules/uswds/dist/img/us_flag_small.png:1
(function (exports, require, module, __filename, __dirname) { �PNG
```